roles/ec2_provision/defaults/main.yml

```diff
@@ -3,3 +3,6 @@ ec2_region: "{{ lookup('ENV', 'EC2_REGION') or 'us-east-2' }}"
 ec2_user: "{{ ansible_user }}"
 nfs_cidr: "10.1.0.0/16"
 nfs_subnet_cidr: "10.1.1.240/28"
+instance_name: nfs-server
```

Comment:
I would consider using ec2_key and ec2_region from config/defaults.yml , since these are common vars for all roles in mig-ci. Duplicated vars can cause confusion... also ec2_user is debatable as well, I believe we use it in other roles as well...